test/cpp/end2end/end2end_test.cc

```diff
@@ -34,10 +34,10 @@
 #include <chrono>
```

Comment:
Aaand in a few other places in fact. Thanks for pointing it out. The tests currently all don't compile under Windows, so it wasn't obvious this was still breaking.